### DISCUSSION TITLE
How to use react, esxjs and vite to create a chrome extension / project template, boiler plate

### DISCUSSION THREAD

**Discussion by coder-abhi07** [community/community#50412] [Other Feature Feedback, Questions, & Ideas]
main.jsx:1 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

i'm getting this error while following all the official documentation of csxjs

**Comment by hasin202** [comment]
Hi. Did you ever figure this out?

**Comment by williambalea** [comment]
I added an answer [here](https://stackoverflow.com/questions/75780036/getting-error-in-create-a-new-project-using-csxjs-vite-and-react-to-create-a-ch).
